examples/local_ratelimit/ratelimit-envoy.yaml

```diff
@@ -78,7 +82,7 @@ static_resources:
               token_bucket:
                 max_tokens: 2
                 tokens_per_fill: 2
-                fill_interval: 5s
+                fill_interval: 2s
```

Comment:
is this correct ? i think it was changed previously to resolve a flake

Reply:
the default rate limit will not be applied except the default `fill_interval` when descriptors are set. It means that the local rate_limit plugin will check if the buckets needs to be filled every default `fill_interval`, so in this example,  the `fill_interval` of descriptors is set 6s, the default `fill_interval` should be set to the factor of 6s such as 2s.